src/makeRobotAccountant.js

```diff
@@ -6,7 +6,20 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let callCount = 0;
+
+  return x => y => {
```

Comment:
not so clear syntax here, it's hard to understand what will be returned from the function